### PR TITLE
fix(route): multiple videos in posts of shuiguopai

### PR DESCRIPTION
**Pull request by nczitzk**
## 该 PR 相关 Issue / Involved Issue

Close #

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/shuiguopai
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [ ] 新的路由 New Route
  - [ ] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/shuiguopai/index.js [hunk 1]
```diff
@@ -48,18 +50,27 @@ module.exports = async (ctx) => {
...
+
+                for (const d of data) {
+                    for (const t of d.tags) {
+                        categories.push(t.tag_title);
+                    }
+                }
 
-                item.author = data.actor;
-                item.category = data.tags.map((t) => t.tag_title);
+                item.category = categories;
```
Comment (TonyRL) [inline lib/v2/shuiguopai/index.js:70]:
```js
item.category = data.flatMap((d) => d.tags.map((t) => t.tag_title));
```

Comment (nczitzk) [reply lib/v2/shuiguopai/index.js:70]: 👍

File: lib/v2/shuiguopai/index.js [hunk 2]
```diff
@@ -6,19 +6,21 @@ const { art } = require('@/utils/render');
 const path = require('path');
 
 module.exports = async (ctx) => {
-    const rootUrl = 'https://shuiguopai.com';
+    const rootUrl = 'http://shuiguopai.com';
```
Comment (TonyRL) [inline lib/v2/shuiguopai/index.js:9]: This domain is dead.

Comment (nczitzk) [reply lib/v2/shuiguopai/index.js:9]: 👌

**Pull request merged**